services/reports-analytics/test/ts/env.influxdb

```diff
@@ -1,5 +1,5 @@
 INFLUXDB_DATA_ENGINE=tsm1
-INFLUXDB_LOGGING_LEVEL=error
+INFLUXDB_LOG_LEVEL=error
```

Comment:
InfluxDB uses "logging" for variable declarations.

https://docs.influxdata.com/influxdb/v1.8/administration/config/#logging-settings